examples/passport/login-with-nextjs/src/app/auth-event-handling/page.tsx

```diff
@@ -0,0 +1,243 @@
+      
+      if (provider) {
+        // Get accounts
+        const accounts = await provider.request({ method: 'eth_requestAccounts' });
+        if (accounts && accounts.length > 0) {
+          setAddress(accounts[0]);
+          setAccountsState(accounts);
+          // Log the accounts change manually since the event might not fire
+          logEvent(ProviderEvent.ACCOUNTS_CHANGED, { accounts });
+        }
+        
+        // Get chain ID
+        const chainId = await provider.request({ method: 'eth_chainId' });
+        setChainId(chainId);
+      }
+      
+      setIsLoggedIn(true);
+    } catch (error) {
+      console.error('Login error:', error);
+      logEvent('login_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  // Handle logout
+  const handleLogout = async () => {
+    try {
+      setLoading(true);
+      await passportInstance.logout();
+      setIsLoggedIn(false);
+      setAddress('');
+      setChainId('');
+      setAccountsState([]);
+    } catch (error) {
+      console.error('Logout error:', error);
+      logEvent('logout_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  // Manually request accounts
+  const requestAccounts = async () => {
+    if (!provider) return;
+    
+    try {
+      setLoading(true);
+      const accounts = await provider.request({ 
+        method: 'eth_requestAccounts' 
+      });
+      logEvent('request_accounts', { accounts });
+      if (accounts && accounts.length > 0) {
+        setAddress(accounts[0]);
+        setAccountsState(accounts);
+      }
+    } catch (error) {
+      console.error('Request accounts error:', error);
+      logEvent('request_accounts_error', error);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <Stack gap="space.large">
+      <Heading>Passport SDK - Event Handling Example</Heading>
+      
+      <Stack gap="space.medium">
+        <Body>Status: {isLoggedIn ? 'Logged In' : 'Logged Out'}</Body>
+        
+        {isLoggedIn && address && (
+          <Stack gap="space.small">
+            <Body weight="bold">Connected Account:</Body>
+            <code>{address}</code>
+          </Stack>
+        )}
+        
+        {isLoggedIn && chainId && (
+          <Stack gap="space.small">
+            <Body weight="bold">Chain ID:</Body>
+            <code>{chainId}</code>
+          </Stack>
+        )}
+        {accountsState && accountsState.length > 0 && (
+          <Stack gap="space.small">
+            <Body weight="bold">Accounts:</Body>
+            <div>
+              {accountsState.map((account, idx) => (
+                <div key={idx} style={{ marginBottom: "4px" }}>
+                  <code>{account}</code> {idx === 0 && <span style={{ fontSize: "12px", color: "#666" }}>(active)</span>}
+                </div>
+              ))}
+            </div>
+          </Stack>
+        )}
+      </Stack>
+      
+      <Stack direction="row" gap="space.large">
```

Comment:
Can you move the buttons to be above the account state data table? Follow the same format as the other apps e.g.

- Heading
- Buttons
- State Data Table
- Event Log